koku/masu/openapi.json

```diff
@@ -812,7 +812,92 @@
         ]
       }
     },
-    "/gcp_invoice_monthly_cost/": {
+    "/db-performance/db-version/": {
+        "get": {
+          "summary": "Returns a table showing the PostgreSQL database software version.",
+          "operationId": "dbPerformanceDbVersion",
+          "description": "Returns a HTML document showing the database version.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing the database version.",
+              "content": "text/html"
```

Comment:
**Issue(blocking):** This does not meet OpenAPI specs: `Structural error at paths./db-performance/db-version/.get.responses.200.content should be object`
**Suggestion:** Update the content field to meet OpenAPI specs

Reply:
I added this because people kept asking about that file.
But: If these endpoints are returning HTML instead of JSON, do they _really_ belong in openapi.json?